UI/EmuScreen.cpp

```diff
@@ -70,10 +70,22 @@
 #include "Windows/MainWindow.h"
 #endif
 
+bool frameStep_;
+
+static void __EmuScreenVblank()
+{
+	if (frameStep_)
+	{
+		frameStep_ = false;
+		Core_EnableStepping(true);
```

Comment:
Oh, in `EmuScreen` this might actually be a better place:

https://github.com/RisingFog/ppsspp/blob/842a29b0aedd2e5db59128f3b25b06d803d27386/UI/EmuScreen.cpp#L980

This will be once per displayed frame though, so importantly:
 * If frameskip is enabled, it will not happen for skipped frames.
 * If the game runs at 30fps, it will be at 30fps.

-[Unknown]